task/tkn-bundle-oci-ta/0.2/MIGRATION.md

```diff
@@ -0,0 +1,6 @@
+# Migration from 0.1 to 0.2
+The parameter `URL` was added to the task.
+The parameter `REVISION` was added to the task.
+
+## Action from users
+Add the `URL` and `REVISION` parameters to the `tkn-bundle-oci-ta` task.
```

Comment:
Does it make sense to describe it in more details? Where a user can get the `URL` and `REVISION`?